jobtastic/task.py

```diff
@@ -157,7 +160,19 @@ def _get_possible_broker_errors_tuple(self):
             # Celery 2.5 uses `broker_connection` instead
             dummy_conn = self.app.broker_connection()
 
-        return dummy_conn.connection_errors + dummy_conn.channel_errors
+        possible_broker_errors = (
+            dummy_conn.connection_errors + dummy_conn.channel_errors
+        )
+        try:
+            from kombu.exceptions import OperationalError
+            # In celery 4.x when a broker is not connected it throws an
+            # OperationalError from kombu. It should also be noted that the
+            # newest version of kombu actually hangs forever. So we need to peg
```

Comment:
what is the newest version of kombu?

Reply:
4.1.0. We are pegging tests to use 4.0.2

Reply:
should we put that version in the comment instead of `newest version`?

Reply:
so whoever works on this next knows what version of kombu was broken from the comment, isntead of digging which one was at the time of this PR